ush/predef_grid_params.yaml

```diff
@@ -877,35 +912,36 @@
 #-----------------------------------------------------------------------
 #
 # Emulation of GSD's RAP domain with ~13km cell size.
+# Note: Close to RRFS_dev1's GSD_RAP13km. Differences are given in comments
 #
 #-----------------------------------------------------------------------
 #
 "RRFS_NA_13km":
```

Comment:
This had a different name in GSL fork. I kept the values in SRW and put the ones in GSL in comments.